app/boards/blackpill_f411ce.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_CONSOLE=n
+CONFIG_SERIAL=n
+CONFIG_UART_CONSOLE=n
+CONFIG_UART_INTERRUPT_DRIVEN=n
+CONFIG_ZMK_USB=y
+CONFIG_ZMK_BLE=n
+CONFIG_ZMK_KSCAN_MATRIX_POLLING=y
+CONFIG_BUILD_OUTPUT_UF2=y
```

Comment:
Is this really the default needed for these? Or are you assuming someone will flash a UF2 bootloader to use ZMK?